docs/sphinx/index.rst

```diff
@@ -154,26 +154,39 @@ copied to destination directory. This file specifies the compiler settings and p
-  package_final_phase  ``--package-final-phase``  Controls after which phase Spack should stop     **None**
-  package_source_dir   ``--package-source-dir``   Controls the source directory Spack should use   **None**
-  spack_url            **None**                   Url where to download Spack                      ``https://github.com/spack/spack.git``
-  spack_commit         **None**                   Spack commit to checkout                         **None**
-  spack_activate       **None**                   Spack packages to activate                       **None**
- ==================== ========================== ================================================ =======================================
+1. `--project.json=[path/to/project.json]` command line option
+2. `project.json` that lives in the same directory as `uberenv.py`
+3. `.uberenv_config.json` found recursively in a parent directory (typically at the root of your project)
 
-However, Uberenv configuration may instead sit in a location external to the uberenv directory itself. In particular when Uberenv is used as a submodule. Uberenv identifies such a case when no ``project.json`` file is found next to the script. It will then look for ``.uberenv_config.json`` recursively in parent directories. The typical usage is to place it at the root directory of the project.
+Project settings are as follows:
 
-When used as a submodule ``.uberenv_config.json`` should define both ``spack_configs_path`` and ``spack_packages_path``, providing Uberenv with the respective location of ``spack_configs`` and ``packages`` directories. Indeed, they cannot sit next to ``uberenv.py`` as per default, since Uberenv repo does not provide them.
+ ========================== ========================== ================================================ =======================================
+  Setting                   Command line Option        Description                                      Default
+ ========================== ========================== ================================================ =======================================
+  package_name              ``--package-name``         Spack package name                               **None**
+  package_version           **None**                   Spack package version                            **None**
+  package_final_phase       ``--package-final-phase``  Controls after which phase Spack should stop     **None**
+  package_source_dir        ``--package-source-dir``   Controls the source directory Spack should use   **None**
+  spack_url                 **None**                   Url where to download Spack                      ``https://github.com/spack/spack.git``
+  spack_commit              **None**                   Spack commit to checkout                         **None**
+  spack_activate            **None**                   Spack packages to activate                       **None**
```

Comment:
Does `spack_configs_path` belong in this table?

Reply:
Added... does this make sense to make this `spack_configs_paths` to be consistent with the packages variable?

Reply:
this would not be as simple as what I did... like what do you do if lives in both, do you take the first or last config?

Reply:
I'm not even sure this is helpful